sklearn/linear_model/base.py

```diff
@@ -271,7 +271,10 @@ def predict(self, X):
             indices = (scores > 0).astype(np.int)
         else:
             indices = scores.argmax(axis=1)
-        return self.classes_[indices]
+        if len(self.classes_) == 1:
```

Comment:
I was getting an `IndexError` in `test_common.test_meta_estimators` as can be seen [here](https://travis-ci.org/scikit-learn/scikit-learn/jobs/105733644). I fixed the `IndexError` by making those changes but `test_mlp` is still failing...